config/config.yaml

```
@@ -0,0 +1,42 @@
+dashboard:
+  host: "0.0.0.0"
+  port: 8501
+  debug: false
+  theme: "light"
+
+logging:
+  level: "INFO"
+  format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+  file_path: "logs/fraud_detection.log"
+  max_bytes: 10485760
+  backup_count: 5 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)